src/components/Exchange/OrdersList.js

```diff
@@ -397,13 +409,23 @@ export default function OrdersList(props) {
       const collateralUSD = getUsd(order.purchaseTokenAmount, order.purchaseToken, true, infoTokens);
 
       const error = getOrderError(account, order, positionsMap);
-
+      const orderTitle = `${order.type === INCREASE ? t`Increase` : t`Decrease`} ${indexTokenSymbol} ${
+        order.isLong ? t`Long` : t`Short`
```

Comment:
The same code is repeated three times, maybe it makes sense to create a function for this?